Index: .gitattributes
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Git Attributes\r\n\r\n* text=auto\r\n*.txt text\r\n\r\n*.png binary\r\n*.ttf binary\r\n*.svg binary\r\n*.pdf binary\r\n*.jp[e]?g binary\r\n\r\n*.py python\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .gitattributes	(revision af10fa965fe197c9b0609794f5ca0f658ce36c83)
+++ .gitattributes	(date 1558567663759)
@@ -2,6 +2,10 @@
 
 * text=auto
 *.txt text
+*.rst text
+*.yaml text
+*.ipy text
+*.ipynb text
 
 *.png binary
 *.ttf binary
@@ -9,4 +13,4 @@
 *.pdf binary
 *.jp[e]?g binary
 
-*.py python
+*.py python
\ No newline at end of file
